torch/_inductor/codegen/cpp.py

```diff
@@ -892,10 +892,20 @@ def constant(val, dtype):
 
     @staticmethod
     def index_expr(expr, dtype):
+        bounds = ValueRanges.unknown()
+        # If this expression does not come from an FX node, we compute its bounds
+        if (
+            fx_node := getattr(V.interpreter, "current_node", None)
+        ) and fx_node.target != "index_expr":
+            bounds = bound_sympy(expr)
```

Comment:
Even if it comes from the fx node, we should compute the bounds with `bound_sympy(expr)` anyway, right?

Reply:
When it comes from the fx_node, we'll forward the already-computed nodes in the `getattr` in CSEProxy